gr-zeromq/lib/base_impl.cc

```diff
@@ -54,6 +54,15 @@ namespace gr {
         delete d_context;
     }
 
+    std::string
+    base_impl::last_address()
+    {
+      size_t addr_len = 256;
+      char addr[addr_len];
+      d_socket->getsockopt(ZMQ_LAST_ENDPOINT, addr, &addr_len);
+      return std::string(addr, addr_len-1);
```

Comment:
Thinking about this, doesn't this form of the ‘basic_string‘ ctor only create a std::string *referring* to the char array? I.e. if the same char buffer is reused, the returned std::string would change?